config/initializers/spotlight_initializer.rb

```diff
@@ -110,13 +110,17 @@
 #   'my-local-config': ->(context) { context.my_custom_data_path(context.current_exhibit) }
 # }
 
-Spotlight::Engine.config.exhibit_themes = %w[default example joyce]
+Spotlight::Engine.config.exhibit_themes = %w[default example dr-joyce-brothers blackprint]
```

Comment:
Maybe we could declare custom_themes and default_themes up here so we don't have to list the themes twice?